warehouse/templates/pages/help.html

```diff
@@ -54,7 +54,6 @@
 {% macro installing() %}{% trans %}How do I install a file (package) from PyPI?{% endtrans %}{% endmacro %}
 {% macro publishing() %}{% trans %}How do I package and publish my code for PyPI?{% endtrans %}{% endmacro %}
 {% macro trove_classifier() %}{% trans %}What's a trove classifier?{% endtrans %}{% endmacro %}
-{% macro yanked() %}{% trans %}What's a "yanked" release?{% endtrans %}{% endmacro %}
```

Comment:
Two concerns here: 
- We're losing the localization (this was raised by @ewdurbin)
- We're also losing links/context from `/help` (let's assume there is at least one external link to this, or that some search engine will deposit users here rather than in the docs).

Given that, I think we should probably leave the FAQ intact, but just have a brief answer that links to the relevant page in the user docs (and likely won't become out of sync).

Reply:
Makes sense, I can revert this and the other removals + tweak the `help.html` to be a short answer that links to the user docs.